examples/aws-rds/postgresql/README.md

```diff
@@ -0,0 +1,211 @@
+# AWS RDS PostgreSQL engine
+
+[PostgreSQL](https://www.postgresql.org/) is an object-relational database management system (ORDBMS) with an emphasis on extensibility and on standards-compliance.
+
+## Introduction
+
+This example is to demonstrate how Kanister can be integrated with AWS RDS instance to protect your data using snapshot backup and restore
+
+## Prerequisites
+
+- Kubernetes 1.10+
+- Kanister controller version 0.21.0 installed in your cluster
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#kanctl)
+
+## Create RDS instance on AWS
+
+RDS instance needs to be reachable from outside world. So make sure that you have VPC with security group having rule to allow ingress traffic on 5432 TCP port.
+
+You can create security group and add rules to defaul VPC using following commands
+
+```bash
+aws ec2 create-security-group --group-name pgtest-sg --description "pgtest security group"
+aws ec2 authorize-security-group-ingress --group-name pgtest-sg --protocol tcp --port 5432 --cidr 0.0.0.0/0
+```
+
+Now create a RDS instance with postgresql engine
+
+```bash
+aws rds create-db-instance \
+    --publicly-accessible \
+    --allocated-storage 20 --db-instance-class db.t2.micro	 \
+    --db-instance-identifier test-postgresql-instance \
+    --engine postgres \
+    --master-username master \
+    --vpc-security-group-id sg-xxxxyyyyzzz \ # Sec group with TCP 5432 inbound rule
+    --master-user-password secret99
+
+aws rds wait db-instance-available --db-instance-identifier=test-postgresql-instance
+```
+
```

Comment:
@PrasadG193 @tdmanv - I'm wondering if we can simplify this top level README. The only requirement for someone to test this out should be:
1) Create a new RDS instance or have instanceID/host info available for an existing instance
2) Create a ConfigMap with this info
3) Create a Blueprint
4) Run backup/restore commands

The pgtest app is useful for testing but complicates this example.

Reply:
Won't that confuse users? They may not get the context behind why do they need configmap and which configmap to use/update. 
In current PR we can add a note suggesting instead of pgtest app, they can use their own application

Reply:
Users in this example likely already have an application that is using an RDS DB and pgtest on the top level will be confusing.

What will be most useful for them will be to know how they can use this example blueprint with their DB instance. 

I think the pgtest example is useful but perhaps this detailed README.md is in a sub-folder and the top level is very minimal

Reply:
@vkamra @tdmanv I have updated the README as per your suggestions.